source/beamline/CLS/CLSSIS3820ScalerContinuousMode.h

```diff
@@ -1,31 +1,25 @@
-#ifndef CLSSIS3820SCALERMODECONTROL_H
-#define CLSSIS3820SCALERMODECONTROL_H
+#ifndef CLSSIS3820SCALERCONTINUOUSMODE_H
+#define CLSSIS3820SCALERCONTINUOUSMODE_H
 
-#include "beamline/AMPseudoMotorControl.h"
+#include "beamline/AMEnumeratedControl.h"
 
-class CLSSIS3820ScalerModeControl : public AMPseudoMotorControl
+class CLSSIS3820Scaler;
+
+class CLSSIS3820ScalerContinuousMode : public AMEnumeratedControl
```

Comment:
I think it would probably be a better idea to go with "ScanningMode" to distinguish between mode and "Input Mode" since Continuous is one of the possible ScanningMode states.